src/openrct2/ride/Ride.h

```diff
@@ -119,7 +119,7 @@ struct Ride
     ObjectEntryIndex subtype;
     RideMode mode;
     uint8_t colour_scheme_type;
-    VehicleColour vehicle_colours[OpenRCT2::Limits::MaxTrainsPerRide + 1];
+    VehicleColour vehicle_colours[OpenRCT2::Limits::MaxVehicleColours];
```

Comment:
Needs import checking. We may need to stick with the old value, even if it’s not quite correct (being 1 too large).